src/main/java/com/sse/ooseproject/controllers/StudentController.java

```diff
@@ -1,18 +1,37 @@
 package com.sse.ooseproject.controllers;
 
+import com.sse.ooseproject.StudentRepository;
+import com.sse.ooseproject.models.Student;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+
 @Controller
 public class StudentController {
-    public StudentController(/* TODO add required repositories. */) {
```

Comment:
Brauchen wir das class for dem StudentController?

Reply:
Adding the class is correct. As the public StudentController is the constructor.